packages/api-explorer/src/components/DocPseudo/DocParams.tsx

```diff
@@ -9,12 +33,12 @@ export interface DocArgsProps {
 }
 
 export const DocParams: FC<DocArgsProps> = ({ parameters }) => (
-  <Flex flexWrap="wrap">
+  <span>
```

Comment:
Does this span actually need to get outputted? If not using a fragment would be more ideal.

```suggestion
  <>
```